contracts/ERC1238/ERC1238.sol

```diff
@@ -163,9 +164,12 @@ contract ERC1238 is ERC165, IERC1238, ERC1238Approval {
         uint8 v,
         bytes32 r,
         bytes32 s,
+        uint256 approvalExpiry,
         bytes memory data
     ) internal virtual {
-        bytes32 messageHash = _getMintApprovalMessageHash(to, id, amount);
+        require(approvalExpiry >= block.timestamp, "ERC1238: invalid approval expiry time");
```

Comment:
The revert message should probably be something like "mint consent has expired" so its more accurate to the error.

Reply:
Fixed in https://github.com/violetprotocol/ERC1238-token/pull/13/commits/d970ec2ef4ea8f462b80d90d6268ce81bf3681d3